src/main/java/core/basesyntax/service/impl/ReportCreatorServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportCreatorService;
+import java.util.Map;
+
+public class ReportCreatorServiceImpl implements ReportCreatorService {
+    private static final String HEADER = "fruit,quantity";
+    private static final String DELIMITER = ",";
+
+    public String createReport(Map<String, Integer> mapFruit) {

```

Comment:
```suggestion
    public String createReport() {
```
You have access to storage